_scripts/update.py

```diff
@@ -16,16 +16,6 @@
 from compare_locales.parser import Entity, getParser
 from compare_locales.paths import ProjectFiles, TOMLParser
 
-HEAD = "master"
```

Comment:
My presumption was that we would only ever be changing this exactly once, when the proper git repo becomes available, so making it easily configurable was unnecessary. But I'm okay with it being in the config JSON.